doc/src/manual/strings.md

```diff
@@ -41,9 +41,10 @@ There are a few noteworthy high-level features about Julia's strings:
 
 ## [Characters](@id man-characters)
 
-A `Char` value represents a single character: it is just a 32-bit primitive type with a special literal
+An `AbstractChar` value represents a single character: it is just a 32-bit primitive type with a special literal
```

Comment:
No longer accurate since an `AbstractChar` could have any representation at all. The simplest fix is just to leave this as `Char`.